skyrim-platform/src/platform_se/skyrim_platform/main.cpp

```diff
@@ -63,8 +64,15 @@ extern CallNativeApi::NativeCallRequirements g_nativeCallRequirements;
 
 void GetTextsToDraw(TextToDrawCallback callback)
 {
-  callback(TextToDraw());
-  callback(TextToDraw());
+  auto text = &TextsCollection::GetSinglton();
+
+  try {
```

Comment:
this try/catch doesn't seem useful. consider removing it. or explain what kind of exceptions does it catch